### ISSUE TITLE
Error weaving POD: Can't call method "notice" on an undefined value

### ISSUE THREAD

**Issue by preaction** [preaction/Statocles#513]
This one comes in from CPAN Testers: http://www.cpantesters.org/cpan/report/37736a9c-7e65-11e6-8efa-05c5ea7e2f12

```
Error weaving POD for path "/tmp/loop_over_bdir-11294-B41gnj/Statocles-0.077-0/t/share/app/perldoc/lib-weaver/My.pm": Can't call method "notice" on an undefined value at /home/sand/src/perl/repoperls/installed-perls/perl/v5.25.4-170-g2c4879e/109d/lib/site_perl/5.25.5/Pod/Weaver/Section/Legal.pm line 56.
```

Does this mean that Pod::Weaver's API changed? Or did we give it some bad input information?

**Comment by eserte** [comment]
Maybe a bleadperl change? I see the error only with perl 5.25.5, but not with earlier perls.

@andk: maybe a git-bisect candidate?

**Comment by eserte** [comment]
Correction: I have now a fail report with perl 5.24.0.

**Comment by eserte** [comment]
Yes, it's quite likely a change in Pod::Weaver 4.014 (@rjbs: FYI):
```
****************************************************************
Regression 'qr:#\s+Pod::Weaver\s+any\s+(\S+)'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    75828566159426976.00
[1='eq_4.012']       -0.0000          0.0000      -2.41
[2='eq_4.013']       -0.0000          0.0000      -7.48
[3='eq_4.014']       -1.0000          0.0000    -64086835319602520.00
[4='eq_missing']              0.0000          0.0000       1.30

R^2= 1.000, N= 47, K= 5
```

**Comment by eserte** [comment]
Similar issue: https://rt.cpan.org/Ticket/Display.html?id=118089

**Comment by preaction** [comment]
I've created a PR for Pod::Weaver to fix this: rjbs/Pod-Weaver#44. When that is released, I will update the version requirements for Statocles.

**Comment by preaction** [comment]
I've released a version (0.078) which skips the tests for Pod::Weaver 4.014 for now (and admonishes people who want weaving to downgrade to 4.013), so at least testers won't be sending in false-positives.
